### DISCUSSION TITLE
Who is using gotd?

### DISCUSSION THREAD

**Discussion by nnqq** [gotd/td#1792] [General]
If you are using **gotd**, add a comment with your company name, URL & logo and we will add it to the Readme. Thank you!

**Comment by nnqq** [comment]
Telegram Questions is the project which parse and save messages from IT-related chats, and shows it as threads on the website https://telq.org

**gotd** helped quickly develop a stable, easily maintained project. Currently **gotd** handles 500 active chats with a total of 30-50 new messages per second.

[<img src="https://user-images.githubusercontent.com/43930873/142857277-a105a47d-12a9-4a66-9a78-43ce81c58b30.png" width="150">](https://telq.org)

**Comment by qiangweihewu** [comment]
> 

Where is this project's source code?

**Comment by nnqq** [comment]
> Where is this project's source code?

@qiangweihewu it is partially open source https://github.com/telq-org
Backend and chat scanner components is not open source

**Comment by SantiiRepair** [comment]
### TGoCall

Site: https://github.com/TGoCall

Description: Library to make Telegram calls 

Status: Work in progress

<img src="https://raw.githubusercontent.com/TGoCall/designs/main/TGoCall.png" />

**Comment by luomoxu** [comment]
https://github.com/tgbot-upnp/tgbot-upnp
a small tool that can cast telegram videos to other devices through the upnp protocol

**Comment by DearRude** [comment]
siahe - an event signup manager and ticketing system in Telegram.
https://github.com/DearRude/siahe

**Comment by iamxvbaba** [comment]
### gramsrv

Repository: https://github.com/iamxvbaba/gramsrv  
Website: https://telesrv.net/

`gramsrv` is an unofficial Telegram-like MTProto server written in Go. We use **gotd/td** as the TL/MTProto protocol foundation while building a self-hosted server focused on real Telegram client compatibility, especially Telegram Desktop and Android client flows.

gotd has been very helpful for keeping the protocol layer reproducible and for iterating on client compatibility research without reinventing the TL/MTProto stack from scratch.
